yorkie/src/test/kotlin/dev/yorkie/JsonTestUtils.kt

```diff
@@ -0,0 +1,15 @@
+// ktlint-disable filename
+
+package dev.yorkie
+
+import com.google.gson.Gson
+import com.google.gson.JsonObject
+import org.junit.Assert.assertEquals
+
+private val gson = Gson()
+
+fun assertJsonContentEquals(expected: String, actual: String) {
```

Comment:
@7hong13 @hackerwins I utilized GSON in test to compare two JSON contents.
this function is replacement of `toSortedJson`.